test_util/marathon.py

```diff
@@ -253,7 +253,7 @@ def ensure_deployment_is_finished():
         except retrying.RetryError:
             raise Exception("Deployments were not completed within {timeout} seconds".format(timeout=timeout))
 
-    def deploy_pod(self, pod_definition):
+    def deploy_pod(self, pod_definition, timeout):
```

Comment:
Every other ```timeout``` in this file has a default value, is this one different for a particular reason?